aiidalab_widgets_base/viewers.py

```diff
@@ -16,6 +16,7 @@
 from aiida.orm import Node
 from ase import Atoms, neighborlist
```

Comment:
Please move this import into the function `compute_bonds`.

Reply:
good catch, I will :)

Reply:
done :)